CONTRIBUTING.md

```diff
@@ -40,30 +40,10 @@ By convention:
 
 ### Test Environment
 
-```bash
-# Setup the virtualenv
-export BG_VENV=bg
-virtualenv ${BG_VENV}
-source ${BG_VENV}/bin/activate
-
-# By default carbon and graphite-web are installed in /opt/graphite,
-# We want NO prefix in order to have a good interaction with virtual env.
-export GRAPHITE_NO_PREFIX=true
-
-# Install Graphite Web and Carbon
-pip install graphite-web
-pip install carbon
+To set-up your test enviroment, source `scripts/env.sh`:
 
-# Install the libffi-dev package from your distribution before running pip install
-pip install -r requirements.txt
-pip install -r tests-requirements.txt
-pip install -e .
-
-# Install Cassandra and Elasticsearch
-export CASSANDRA_VERSION=3.11.2
-export ES_VERSION=6.3.1
-export CASSANDRA_HOME=$(pwd)/apache-cassandra-${CASSANDRA_VERSION}
-./tools/install-deps.sh
```

Comment:
I kind of like the manual instructions because it explained how things work.. We can see if we re-add them later.